test/WebSites/BasicWebSite/Controllers/ApplicationBaseController.cs

```diff
@@ -1,10 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-namespace BasicWebSite.Controllers
-{
-    public abstract class ApplicationBaseController
```

Comment:
Could keep this by using `[Controller]`. I agree though, not a super useful functional test